docs/erc1155-skills.md

```diff
@@ -158,17 +161,6 @@ To update the SOEF config, run in each AEA project:
 aea config set vendor.fetchai.connections.soef.config.chain_identifier ethereum
 ```
 
-## Run Ganache
-
-Run the following command
-```
-docker run -p 8545:8545 trufflesuite/ganache-cli:latest --verbose --gasPrice=0 --gasLimit=0x1fffffffffffff
---account="$(cat erc1155_deployer/ethereum_private_key.txt),1000000000000000000000"
---account="$(cat erc1155_client/ethereum_private_key.txt),1000000000000000000000"
```

Comment:
with copy&paste, this didn't work - it would spawn the Ganache image without `--account` arguments

Reply:
ok